crates/bitwarden/src/client/access_token.rs

```diff
@@ -15,6 +15,15 @@ pub struct AccessToken {
     pub encryption_key: SymmetricCryptoKey,
 }
 
+// We don't want to log the more sensitive fields from an AccessToken
+impl Debug for AccessToken {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.debug_struct("AccessToken")
+            .field("access_token_id", &self.access_token_id)
+            .finish()
```

Comment:
This only logs the id?

Reply:
Yeah, I didn't want to log the secret or the encryption key to avoid leaking potentially sensitive information in the logs. I think just logging the ID is a good compromise between debug-ability and security in this case.